modules/javafx.controls/src/test/java/test/javafx/scene/control/CellTest.java

```diff
@@ -367,19 +369,24 @@ public void selectingAnEmptyCellResultsInNoChange() {
         assertEquals("editable", cell.editableProperty().getName());
     }
 
-    // When the cell was focused, but is no longer focused, we should cancel editing
-    // Check for focused pseudoClass state change?
-    @Ignore(value = "I'm not sure how to test this, since I need a scene & such to move focus around")
     @Test public void loseFocusWhileEditing() {
         Button other = new Button();
         Group root = new Group(other, cell);
         Scene scene = new Scene(root);
 
+        Stage stage = new Stage();
+        stage.setScene(scene);
+        stage.show();
+        stage.requestFocus();
+        Toolkit.getToolkit().firePulse();
+
         CellShim.updateItem(cell, "Apples", false);
         cell.startEdit();
         cell.requestFocus();
+        Toolkit.getToolkit().firePulse();
 
```

Comment:
Minor: Optional:
Adding `assertTrue(cell.isEditing());` here can be a good sanity check. But I leave it to you.
Similar comment for the change in `DateCellTest.loseFocusWhileEditing()`

Reply:
Done.